website/src/components/Metadata/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import {Helmet} from "react-helmet";
+import { getHostOrigin } from "../../utils/getHostOrigin";
+import { articleSchema } from "./articleSchema";
+import { websiteSchema } from "./websiteSchema";
+import { Author } from "./metadata.models";
+
+
+interface MetadataProps {
+    title: string;
+    description: string;
+    url: string;
+    image: string;
+    type?: string;
+    isPost: boolean;
+    tags?: [string];
+    author?: Author;
+}
+
+
+export const Metadata: React.FC<MetadataProps> = ({ title, description, url, isPost , image, tags, author, type }) => {
+    
+    const site = {
+        logo: `${getHostOrigin}/images/png/logo.png`,
+        siteUrl: `${getHostOrigin}`
+    }
+    let imageObj = {
+        src: image,
+        shareImageWidth: '1000px',
+        shareImageHeight: '523px'
```

Comment:
Recommended image size is 1200 x 630. Can we use the same? https://developers.facebook.com/docs/sharing/webmasters/images/